docs/WSL2_AI_PERFORMANCE_TUNING.md

```
@@ -0,0 +1,268 @@
+2. Ensure `[gpu] enabled=true` is set in `/etc/wsl.conf`.
+3. Verify GPU availability inside WSL2:
+   ```bash
+   nvidia-smi
+   ```
+4. Supports PyTorch, TensorFlow, DirectML, and local LLM runners (Ollama / vLLM / llama.cpp).
+
+---
+
+## 8. Security & Environment Variable Management
+
+### Protecting API Keys
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)